tools/ppchcalls.py

```diff
@@ -94,11 +94,12 @@
     420: 'H_CREATE_RPT',
     424: 'H_REMOVE_RPT',
     428: 'H_REGISTER_RPAGES',
-    432: 'H_DISABLE_AND_GETC',
+    432: 'H_DISABLE_AND_GET',
```

Comment:
This could have been a separate patch, with appropriate commit log.

Reply:
Done, included a separate patch for fixing this.